content/quickstart.md

```diff
@@ -73,22 +73,29 @@ $ export GITHUB_CLIENT_ID=...
 $ export GITHUB_CLIENT_SECRET=...
 ```
 
-Install the OpenShift template (syndesis-dev.yml or syndesis.yml as discussed [above]({{< relref "#template-selection" >}})):
+Install the OpenShift template (syndesis-restricted-dev.yml or syndesis-restricted.yml as discussed [above]({{< relref "#template-selection" >}})):
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/syndesis-dev.yml
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/syndesis-restricted-dev.yml
 ```
 
-Deploy syndesis using the following command, replacing "syndesis-dev" with "syndesis" depending on the template
+In order to make it easy to run Syndesis on a cluster without requiring admin rights, Syndesis takes advantage of OpenShift's ability to use a [Service Account as an OAuth client](https://docs.openshift.org/latest/architecture/additional_concepts/authentication.html#service-accounts-as-oauth-clients). Before we create the app, we'll need to create this Service Account:
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/support/serviceaccount-as-oauthclient-restricted.yml
+```
+
+Deploy syndesis using the following command, replacing "syndesis-restricted-dev" with "syndesis-restricted" depending on the template
 you have just installed:
 
 ```bash
-$ oc new-app syndesis-dev \
+$ oc new-app syndesis-dev-restricted \
     -p ROUTE_HOSTNAME=syndesis.$(minishift ip).xip.io \
     -p OPENSHIFT_MASTER=$(oc whoami --show-server) \
+    -p OPENSHIFT_PROJECT=$(oc project -q) \
+    -p OPENSHIFT_OAUTH_CLIENT_SECRET=$(oc sa get-token syndesis-oauth-client) \
     -p GITHUB_OAUTH_CLIENT_ID=${GITHUB_CLIENT_ID} \
-    -p GITHUB_OAUTH_CLIENT_SECRET=${GITHUB_CLIENT_SECRET} \
-    -p INSECURE_SKIP_VERIFY=true
+    -p GITHUB_OAUTH_CLIENT_SECRET=${GITHUB_CLIENT_SECRET}
 ```
```

Comment:
Are cert fixed on Minishift now? Will it work without `INSECURE_SKIP_VERIFY=true`?